modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -220,6 +220,96 @@ def snapshot_create(self, snapshot_name: str, timeout_sec: int = 60):
         raise TimeoutError(f'Failed to wait for the snapshot operation to complete: '
                            f'snapshot_name={snapshot_name} in {timeout_sec} seconds.')
 
+    def incremental_snapshot_create(self, snapshot_name: str, exp_increment: int, timeout_sec: int = 60):
+        """
+        Create snapshot.
+        :param snapshot_name: Name of Snapshot.
+        :param exp_increment: Expected increment index.
```

Comment:
Can we do not require the caller of the method to know the extepcted increment index? Looks like it may be annoying for the caller.

Can we instead say request the current index before the snapshot command invokation and return new index as a return value upon the operation completion?

And lets the caller decide if it is expected or not if it wants to.

Reply:
I suggest to remove increment index at all. Test later tries to restore snapshot with specific index. I think it's enough to validate that snapshot was actually created.